docs/admin/install.md

```diff
@@ -13,9 +13,9 @@ Here you will find detailed instructions on how to **install** and **configure**
 ## Install Guide
 
 !!! note
-    Plugins can be installed manually or using Python's `pip`. See the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this plugin is [`nautobot-floor-plan`](https://pypi.org/project/nautobot-floor-plan/).
+    Apps can be installed manually or using Python's `pip`. See the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package) for more details. The pip package name for this app is [`nautobot-floor-plan`](https://pypi.org/project/nautobot-floor-plan/).
```

Comment:
Should `nautobot.readthedocs.io` URL be changed to `docs.nautobot.com`?

Do we need an upstream (core) change to move the URL path to `/apps/` or where is that controlled?

Reply:
I would say yes for both, the latter I think we need to plan and execute. In an ideal world the former would be handled via drift manager.

Reply:
`nautobot.readthedocs.io` should always be replaced with `https://docs.nautobot.com/projects/core/` although it's not breaking anything (RTD redirects it).

Is this a leftover from the old template? The current cookie should not have any `nautobot.readthedocs.io` in it.

Reply:
Btw `https://nautobot.readthedocs.io/en/latest/plugins/#install-the-package` redirects to `https://docs.nautobot.com/projects/core/en/latest/development/apps/#install-the-package`.

Reply:
Yes it came in with the original cookie bake